code/web/src/modules/survey/Survey/Item.js

```diff
@@ -29,7 +29,7 @@ class Item extends PureComponent {
 
   handleItemClick = () => {
     this.setState({ selected: !this.state.selected });
-    this.props.updateSelectedItems(this.props.item, this.state.selected)
+    this.props.updateSelectedItems(this.props.item, !this.state.selected, this.props.page)
```

Comment:
Thanks for getting this right again. The broken code is persistently sneaking its way into PRs. Let's be mindful of that!